app/models/rook.rb

```diff
@@ -1,3 +1,18 @@
 # frozen_string_literal: true
 class Rook < Piece
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def valid_move?(destination_row, destination_col)
+    return false if diagonal?(destination_row, destination_col)
+    return false if obstructed?(destination_row, destination_col)
+    return false if same_color?(destination_row, destination_col)
+    # rubocop:enable Metrics/CyclomaticComplexity
+
+    return true if empty_or_diff_color?(destination_row, destination_col) && (horizontal?(destination_row, destination_col) || vertical?(destination_row, destination_col))
```

Comment:
since this is the last line in the method. you could leave out ` return true if ` and it would still work as expected :)